wlauto/workloads/googleslides/__init__.py

```diff
@@ -0,0 +1,147 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import os.path as path
+import re
+from wlauto import AndroidUiAutoBenchmark, Parameter
+from wlauto.utils.types import list_of_strings
+
+
+def log_method(workload, name):
+    workload.logger.info('===== {}() ======'.format(name))
+
+class GoogleSlides(AndroidUiAutoBenchmark):
+
+    name = 'googleslides'
+    package = 'com.google.android.apps.docs.editors.slides'
+    description = 'Creates a Google Slides presentation with some commonly used features'
```

Comment:
Can you fill out the description with all of the steps performed by the workload?  We plan to take this text directly into the final report so it should follow the format established in previous workloads.